kalliope/core/Cortex.py

```diff
@@ -113,8 +113,8 @@ def save_parameter_from_order_in_memory(cls, order_parameters):
                 if Utils.is_containing_bracket(value):
                     # if the key exist in the temp dict we can load it with jinja
                     value = jinja2.Template(value).render(Cortex.temp)
-                    if value:
-                        Cortex.save(key, value)
-                        order_saved = True
+                if value:
```

Comment:
Can you please add a unittest on this one so we do not miss it again :) 
link : https://github.com/kalliope-project/kalliope/issues/529